validation/build_validation_databases.R

```diff
@@ -0,0 +1,63 @@
+# To use this file, run it as an R script in the validation directory
```

Comment:
Suggestion that things should be run from the tool directory since it work best with RStudio and most paths in other scripts are relative to the tool directory